pkg_resources/__init__.py

```diff
@@ -1313,13 +1313,15 @@ def get_default_cache():
+
```

Comment:
I really *really* hope this is not the right way to do this. I certainly don't think it should be all on one line.